src/thunderbots/software/ai/hl/stp/tactic/shoot_goal_tactic.cpp

```diff
@@ -25,14 +26,20 @@ std::string ShootGoalTactic::getName() const
 }
 
 void ShootGoalTactic::updateParams(const Field &field, const Team &friendly_team,
-                                   const Team &enemy_team, const Ball &ball,
-                                   std::optional<Point> chip_target)
+                                   const Team &enemy_team, const Ball &ball)
 {
     this->field         = field;
     this->friendly_team = friendly_team;
     this->enemy_team    = enemy_team;
     this->ball          = ball;
-    this->chip_target   = chip_target;
+}
+
+void ShootGoalTactic::updateParams(const Field &field, const Team &friendly_team,
+                                   const Team &enemy_team, const Ball &ball,
+                                   std::optional<Point> chip_target)
```

Comment:
Could we add at least one unit test for this function?

Reply:
We typically don't test the `updateParams` for tactics since no logic is being performed and we would need to add lots of getters. We should talk more about if this is something we want to push for.

Reply:
Sounds good. We'll be changing this up when we change how tactics get updated anyhow